### PR TITLE
Add buster-v4l2 and buster-igt configs

### PR DESCRIPTION
**Pull request by Lakshmipathi**
Include buster-v4l2 and buster-igt rootfs configurations

Signed-off-by: Lakshmipathi Ganapathi <lakshmipathi.ganapathi@collabora.com>

### CODE REVIEW

File: rootfs-configs.yaml [hunk 1]
```diff
@@ -3,14 +3,14 @@ rootfs_configs:
     rootfs_type: debos
     debian_release: buster
     arch_list:
-      - 'amd64'
```
Comment (gctucker) [inline rootfs-configs.yaml]: One patch for this, it's editing the `buster` rootfs config.

File: rootfs-configs.yaml [hunk 2]
```diff
@@ -32,3 +32,58 @@ rootfs_configs:
       - partx
       - find
     script: ""
+  buster-v4l2:
```
Comment (gctucker) [inline rootfs-configs.yaml]: One patch for this one, to add v4l2.  Also please include a blank line between rootfs configs for readability.

File: rootfs-configs.yaml [hunk 3]
```diff
@@ -32,3 +32,58 @@ rootfs_configs:
...
+      - amd64
+      - arm64
+      - armhf
+    extra_packages_remove:
+      - bash
+      - e2fslibs
+      - e2fsprogs
+    script: "scripts/buster-v4l2.sh"
+    test_overlay: "overlays/v4l2"
+  buster-igt:
```
Comment (gctucker) [inline rootfs-configs.yaml]: And a third patch for `buster-igt`, with a blank line too.

**Comment by Lakshmipathi** [comment]
Passed on staging server.

**Comment by gctucker** [comment]
We need to test and merge the first PR with the regular Buster rootfs first.

**Comment by Lakshmipathi** [comment]
Tested on staging with LAVA job:

buster-v4l2 (amd64) - https://lava.collabora.co.uk/scheduler/job/2343173
buster-igt (armhf) - https://lava.collabora.co.uk/scheduler/job/2343601
buster-igt (arm64) - https://lava.collabora.co.uk/scheduler/job/2344391

**Comment by gctucker** [comment]
github.com was down this afternoon, the next staging job should now have URLs with file systems built using the latest version of this PR with the jobs created by `kernelci-jenkins` as per #371.

**Pull request merged**